code/rounds/PostRound.cs

```diff
@@ -60,5 +64,17 @@ protected override void OnStart()
                 }
             }
         }
+
+        private bool CheckMinimumPlayers()
+        {
+            if (Client.All.Count < TTTReborn.Gamemode.Game.TTTMinPlayers)
+            {
+                TTTReborn.Gamemode.Game.Instance.ChangeRound(new WaitingRound());
+
+                return false;
+            }
+
+            return true;
+        }
```

Comment:
This method should be named appropriate for what it does

If I were to read it, I would assume that it would just check for minimum players, but it also ends up changing the round.

We should split this functionality up into two different methods imo.

Reply:
Yea good idea, any suggestions? :D

Reply:
Let me think on that

Reply:
We could use a base class property "HasEnoughPlayers" and a "CheckHasEnoughPlayers" method that does the round switching logic based on the property.